packages/block-editor/src/components/inserter/blocks-types-tab.native.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
Could we rename this to `block-types-tab.native.js` so it matches its web version?

Reply:
I've renamed this component in this [commit](https://github.com/WordPress/gutenberg/pull/28495/commits/08f45490fbc8f16be4957c783d13bf18c2618e6e).